templates/blog.html

```diff
@@ -1,8 +1,15 @@
 {% extends "base.html" %}
-    {% block title %}
-		<title>Blog</title>
-    {% endblock %}
+{% block title %}
+	<title>WhoTracks.me | Blog</title>
+{% endblock %}
 
+{% block og_params %}
+    <!-- OG params for sharable content -->
+    <meta property="og:title" content="WhoTracks.me | Blog">    
+    <meta property="og:type" content="website">
+    <meta property="og:url" content="https://whotracks.me/blog">
+    <meta property="og:description" content="This is our tech blog. Here we will be discussing privacy technlogies, dissecting trackers, map the tracking landscape in websites and write about privacy-preserving engineering.">
```

Comment:
`technologies` -> `technologies`. Can we have that injected into the template from Python maybe? All the metadata could be defined in a central place and injected in templates.